docker/src/main/docker/docker-compose.yml

```diff
@@ -28,13 +28,16 @@ services:
       - cluster.name=contextElasticSearch
     ports: # Expose Elasticsearch ports
       - "9200:9200"
-
+    networks:
+      - unomi
   unomi:
     build: .
     image: apache/unomi:${project.version}
     container_name: unomi
     environment:
       - UNOMI_ELASTICSEARCH_ADDRESSES=elasticsearch:9200
+      - KAFKA_BROKERS=kafka:9093
```

Comment:
It should be still optional. I would rather add another docker-compose with Kafka (lot of users don't care about Kafka for now).